runtime/handler.go

```diff
@@ -137,6 +138,17 @@ type responseBody interface {
 	XXX_ResponseBody() interface{}
 }
 
+// GetResponseBody will return the field that was configured in the "response_body" annotation for this specific
+// response type. The main use of this is if the [WithForwardResponseRewriter] ServeMuxOption is used, those custom
+// response rewriters can still rely on grpc-gateway to understand how to retrieve the configured response body by
+// calling this.
+func GetResponseBody(msg proto.Message) interface{} {
+	if rb, ok := msg.(responseBody); ok {
+		return rb.XXX_ResponseBody()
+	}
+	return msg
+}
```

Comment:
I'm not convinced this needs to be part of the PR. It's a convenience function that could be added at a later stage if necessary, IMO.

Reply:
That is fine. I can remove it. I think my thought process here was "Is `XXX_ResponseBody()` internal to `grpc-gateway`?" — If it needed to be changed for some reason, would that count as a SemVer major change?